src/main/java/io/github/nucleuspowered/nucleus/modules/admin/commands/BlockZapCommand.java

```diff
@@ -39,14 +39,15 @@
 
     @Override public CommandResult executeCommand(CommandSource src, CommandContext args) throws Exception {
         Location<World> location = args.<Location<World>>getOne(this.locationKey).get();
+        String getBlockZapped = location.getBlockType ().getName ();
         if (location.getBlockType() == BlockTypes.AIR) {
```

Comment:
Check this before getting the block as an item and creating text to represent it.